src/encoded/tests/test_schema_library.py

```diff
@@ -28,6 +28,34 @@ def library_starting_quantity_no_units(library):
     })
     return item
 
+@pytest.fixture
+def library_fragmentation_method_string(library):
```

Comment:
If this need to be fixture can we rename to something that indicates it is invalid?

Reply:
done